MultiSource/Benchmarks/Ptrdist/anagram/anagram.c

```diff
@@ -137,6 +137,13 @@
 #include <setjmp.h>
 #include <unistd.h>
 
+jmp_buf jbAnagram;
+
+void *memset(void *s : byte_count(n), int c, size_t n) : bounds(s, (char *) s + n);
```

Comment:
You shouldn't need this, as `stdlib_checked.h` should include a compatible definition that you can use in a checked scope.